clusterloader2/testing/load/config.yaml

```diff
@@ -3,18 +3,18 @@
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
 #Constants
-{{$BIG_GROUP_SIZE := 250}}
-{{$MEDIUM_GROUP_SIZE := 30}}
-{{$SMALL_GROUP_SIZE := 5}}
-{{$NODES_PER_NAMESPACE := 100}}
-{{$PODS_PER_NODE := 30}}
-{{$TEST_THROUGHPUT := 10}}
+{{$BIG_GROUP_SIZE := DefaultParam .BIG_GROUP_SIZE 250}}
+{{$MEDIUM_GROUP_SIZE := DefaultParam .MEDIUM_GROUP_SIZE 30}}
+{{$SMALL_GROUP_SIZE := DefaultParam .SMALL_GROUP_SIZE 5}}
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
+{{$TEST_THROUGHPUT := DefaultParam .TEST_THROUGHPUT 10}}
 # bigRCs - 1/4 of all pods should be in big RCs.
-{{$BIG_RCS_PER_NAMESPACE := 3}}
+{{$BIG_RCS_PER_NAMESPACE := DefaultParam .BIG_RCS_PER_NAMESPACE 3}}
```

Comment:
Also neither of those.

Reply:
I'm not convinced.
`BIG_RCS_PER_NAMESPACE` is a value that depends on `BIG_GROUP_SIZE`, `NODES_PER_NAMESPACE` and `PODS_PER_NODE`. Assuming that we allow to modify any of those, `BIG_RCS_PER_NAMESPACE` should also be modifiable.

Reply:
OK - I see your point.
So maybe this should actually not be hardcoded by switched to: "POD_PER_NODE * Nodes / 1000" (and similarly for the below ones).

Reply:
Done.
However, I changed assumption from "all pods / 4" to "namespace / 4", due to the fact, that we want to have identical namespaces.